config/environment.js

```diff
@@ -5,7 +5,7 @@ module.exports = function(environment) {
     modulePrefix: 'realworld-starter-kit',
     environment,
     rootURL: '/',
-    locationType: 'auto',
+    locationType: 'history',
```

Comment:
Needed to change to `history` because having the `author` route path as `profile/:id` ended up with it appearing as `#profile/23` in the URL rather than `profile/23`